src/app/home-page/widget/entry-boxes/entry-boxes.component.html

```diff
@@ -0,0 +1,63 @@
+<mat-card class="h-100 entry-boxes">
+  <div id="header-bar" fxLayout="row" fxLayoutGap="5px" class="mat-card-header-separate">
+    <div id="workflow-number" fxLayout="row">
+      <mat-card-header fxLayout="row">
+        <mat-card-title>{{ entryType }}s</mat-card-title>
+        <span class="bubble">{{ listOfEntries.length }}</span>
+      </mat-card-header>
+    </div>
+    <button
+      mat-button
+      class="private-btn small-btn-structure ml-3 mb-3"
+      id="register{{ entryType }}Button"
+      matTooltip="Register {{ entryType }}"
+      matTooltipPosition="after"
+      (click)="showRegisterEntryModal()"
+    >
+      <img src="../../../assets/svg/icons-plus.svg" alt="Register a {{ entryType }}" class="pr-2 pb-1" />
+      Register a {{ entryType }}
+    </button>
+  </div>
+  <mat-divider></mat-divider>
+  <app-loading [loading]="isLoading" [retainContent]="true" class="h-100"></app-loading>
+  <mat-card-content class="p-3 entries-container">
+    <div *ngIf="(listOfEntries && listOfEntries.length !== 0) || isLoading; else noEntries">
+      <mat-form-field appearance="outline" class="search-form-field w-100">
+        <input matInput [(ngModel)]="filterText" (keyup)="onTextChange($event)" placeholder="Find {{ entryType }}s..." />
+      </mat-form-field>
+      <span class="weight-bold">Recent Activity</span>
+      <div *ngFor="let entry of listOfEntries" fxLayout="row" class="list-items w-100">
+        <a class="no-wrap" [routerLink]="allEntriesLink + entry.path">
+          {{ entry.prettyPath }}
+        </a>
+        <div class="date">{{ entry.lastUpdateDate | date: 'MMM d, yyyy' }}</div>
+      </div>
+      <a class="weight-bold" [routerLink]="allEntriesLink">
+        All {{ entryType }}<mat-icon class="mt-1">keyboard_double_arrow_right</mat-icon>
```

Comment:
This shows up as `All workflow` for me (missing the "s").

Reply:
oh yes!